packages/scene-composer/src/components/toolbars/common/__snapshots__/ItemContainer.spec.tsx.snap

```diff
@@ -317,6 +315,47 @@ exports[`ItemContainer should render properly and trigger onClick when clicking
 </div>
 `;
 
+exports[`ItemContainer should render selected item properly with custom menuHeight 1`] = `
+.c0 {
+  -webkit-flex: none;
+  -ms-flex: none;
+  flex: none;
+  position: relative;
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-pack: start;
+  -webkit-justify-content: flex-start;
+  -ms-flex-pack: start;
+  justify-content: flex-start;
+  min-width: 40px;
+  height: 100px;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  cursor: default;
+  pointer-events: none;
+  background-color: colorBackgroundItemSelected;
+  border-top: 1px solid colorBorderDividerDefault;
+}
+
+.c0:first-of-type {
+  border-top: 0;
+}
```

Comment:
we use https://www.npmjs.com/package/jest-styled-components to avoid snapshot issue

Reply:
That doesn't fix the snapshot issue, because we see the individual styles right here in the snapshot. This doesn't help customers of our components when writing their own tests. We still create a situation where us changing styles can break previously passing tests downstream that would otherwise pass. It forces customers to effectively adopt styled-components even if they aren't using them.